deploy/operator/crd/healthcheck-crd.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: healthchecks.aspnetcore.ui
+
+spec:
+  group: aspnetcore.ui
+  names:
+    plural: healthchecks
+    singular: healthcheck
+    kind: HealthCheck
+    listKind: HealthChecks
+    shortNames:
+      - hc
+  versions:
+    - name: v1
+      served: true
+      storage: true
+
+  scope: Namespaced
+
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            name:
+              type: string
+            portType:
+              type: string
+              pattern: ^(LoadBalancer|ClusterIP|NodePort)$
+            portNumber:
+              type: number
```

Comment:
Defaulting to `80` seems like an intelligent choice for the port number.

Reply:
Only name and serviceLabel is mandatory now. All the other stuff has internal defaults until 1.17 cluster versions are wide spreaded.